packages/terra-application-utility/src/terra-dev-site/test/application-utility/DefaultMenuUtilityMenu.test.scss

```diff
@@ -0,0 +1,15 @@
+@import './ApplicationUtilityTestCommon';
```

Comment:
we can still use the above DefaultHeaderUtilityMenu.test.scss file i guess. as both are having the same code. we can create the common file as UtilityMenuCommon.scss

Reply:
Agreed, "DefaultMenuUtilityMenu.test.scss" and "DefaultHeaderUtilityMenu.test.scss" were identical in every manner. So, renamed one file to "UtilityMenuCommon.scss" and deleted the other. Also changed the references for the respective files.